src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -161,4 +192,69 @@ public String toString() {
 		return String.format("%s[%s]",
 			getClass().getSimpleName(), Bytes.toBase64String(getPublicKey().getBytes()));
 	}
+
+	// ###  From Client Library  ###
```

Comment:
I don't think these comments add any value in the source. If anything, you can make them here to explain to reviewers, but future maintainers don't need to know this

Reply:
to be removed